secret.go

```diff
@@ -207,3 +298,142 @@ func readSecretFromPath(filePath string, secretSpec HarnessSecret) (*bytes.Buffe
 	}
 	return payload, writer.FormDataContentType(), nil
 }
+
+func createSSHSecret(filepath string, secretIdentifier string, baseURL string, port int, username string, requiresFile bool) error {
+	var err error
+	var secretBody HarnessSecret
+
+	isSSHFileSecret := secretIdentifier == ""
+	if isSSHFileSecret {
+		secretIdentifier = SSH_KEY_FILE_SECRET_IDENTIFIER
+	}
+	createUrl := "v2/secrets"
```

Comment:
Let's declare the endpoints in the constants on the similar lines to other endpoints